bot/cogs/grades_data/grades_cog.py

```diff
@@ -168,9 +198,9 @@ def getFirstLast(self, Name):
         Last = fml[0]
         return First + ' ' + Last
 
-    def process_profQuery(self, Name):
+    def process_profQuery(self, Name, year=2014):
```

Comment:
lets pull this magic number out into a module constant called MIN_YEAR and use that, just easier to keep track of that way and avoids magic numbers